src/components/checkboxTree/utils.ts

```diff
@@ -1,10 +1,11 @@
 import { Composition } from "./types";
+import * as _ from "lodash";
```

Comment:
`import cloneDeep from "lodash/cloneDeep";`
will only import whats needed

Reply:
Maybe we could just install a function module from npm?
https://www.npmjs.com/package/lodash.clonedeep
https://www.npmjs.com/package/@types/lodash.clonedeep